src/egon/data/datasets/heat_supply/individual_heating.py

```diff
@@ -1467,58 +1488,61 @@ def determine_hp_cap_buildings_eGon100RE():
 
     """
 
-    # ========== Register np datatypes with SQLA ==========
-    register_adapter(np.float64, adapt_numpy_float64)
-    register_adapter(np.int64, adapt_numpy_int64)
-    # =====================================================
+    if "eGon100RE" in config.settings()["egon-data"]["--scenarios"]:
```

Comment:
wouldn't it be slightly nicer to insert this 'if' where the function is called? Mhh I am unsure - at first it seemed weird to me to put this if statement in the beginning of the function and put everything inside - but may be it does not matter really.